cypress/utils/dashboards/datasource-management-dashboards-plugin/commands.js

```diff
@@ -155,3 +161,84 @@ Cypress.Commands.add(
     });
   }
 );
+
+Cypress.Commands.add(
+  'selectFromDataSourceSelectable',
+  (dataSourceTitle, dataSourceId) => {
+    cy.get('#dataSourceSelectableContextMenuPopover').click();
+    cy.checkDataSourceExist(DEFAULT_DS_TITLE);
```

Comment:
creating data source with API using cypress util commands won't set as a default data source. I suggest we remove this line, to avoid confusion. While we can still keep the `checkDataSourceExist()` util commands, in case anyone knows what they are doing, and create data source from UI, and then check default exists

```suggestion

```